upup/pkg/fi/cloudup/openstacktasks/instance.go

```diff
@@ -97,30 +100,55 @@ func (e *Instance) Find(c *fi.Context) (*Instance, error) {
 	if e == nil || e.Name == nil {
 		return nil, nil
 	}
+	tmp := strings.Replace(strings.ToLower(fi.StringValue(e.GroupName)), "_", "-", -1)
```

Comment:
Possibly using [strings.Replacer](https://pkg.go.dev/strings?tab=doc#Replacer) makes the code more clear and removes the need for a temporary variable

Reply:
As a style point, I try to put kops specific logic in the model, and keep the tasks generic.  So in this case I would expect to normalize the groupName in the model, and pass it in here as "namePrefix" (or something like that).